docs/_website/generator/generate.go

```diff
@@ -148,6 +148,14 @@ func main() {
 		log.Fatal(err)
 	}
 
+	// Get all images files to be moved.
+	images, err := getFiles(docsRoot, ".png")
```

Comment:
this results in a fatal error when the png is fed to generate then ParseFiles

Reply:
Idk its been working on my feature branch with my blog post.